docs/architecture/adr-008-CERT-group.md

```diff
@@ -0,0 +1,92 @@
+# ADR 008: Computer Emergency Response Team (CERT) Group
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
+information in an emergency scenario, we propose the creation of a
+specialization group named The Computer Emergency Response Team (CERT).
+Initially this group's role is intended to serve as coordinators between the
+validators, the developers, and other members of the wider community.
+Additionally, a special privilege is proposed for the CERT group: the capacity
+to "circuit-break" (aka. temporarily disable)  a particular transaction path. 
```

Comment:
CERT group shouldn't be allowed to disable governance tx's correct? Otherwise they can prevent revocation by governance vote.

This exception should be stated explicitly if it exists

Reply:
I disagree - if there was a problem with governance voting (for instance a capability to vote many times) then governance would be broken and should be halted, it would be then up to the validator set to coordinate and hard-fork upgrade to a patched version of the software where governance is re-enabled (and fixed). If the dCERT group abuses this privilege they should all be fully slashed probably. 

Does this make sense to you? if so I'll add this discussion into the ADR

Reply:
(added my points, however, subject to more discussion still!)

Reply:
Yea I think that makes sense. So if a malicious dCERT group disables governance and then tries to abuse their priveleges without getting overruled by community we hard-fork and slash the group?

Will the hard-fork have to fork from before governance got disabled? If so what are the implications of this for tendermint's finality guarantees?

Reply:
effectively correct, in this context unrightly disabling governance is a slashable offence on its own.

Reply:
We should probably make it fairly difficult to hit a circuit breaker anyways, for instance you need 100% consensus within the dCERT group